### ISSUE TITLE
Content access by path - documentation

### ISSUE THREAD

**Issue by willguv** [localgovdrupal/localgov#638]
Work in progress: This module adds access control to your LGD site sections, which allows you to restrict editing/publishing/etc actions to subsets of users.

https://github.com/localgovdrupal/localgov_content_access_control

**Comment by willguv** [comment]
Video by @markconroy 

https://www.youtube.com/watch?v=wUG7njGXAIU

**Comment by willguv** [comment]
New module - content access by path
https://youtu.be/LboXz5b72XY?si=vfOel3GGhMEH54sC

**Comment by willguv** [comment]
@markconroy can we talk about adding media paths to this? How much effort please - thanks

**Comment by markconroy** [comment]
@willguv media doesn't have paths by default - media URLs are just /media/123 but site visitors should be be able to visit those URLs, they are only for admins to edit media.

For that reason, I don't think this media is suitable for "content access by path". Especially since media should be shared across a site (that's the whole point of media library). If we gave media a URL of say /media/fostering/hero.jpg, it's very conceivable that the homepage or some other page would also want to use that image as a teaser in a component.

**Comment by willguv** [comment]
@markconroy have you pushed this back to LGD? No worries if not, just seeing where we've got to

**Comment by markconroy** [comment]
Do you mean to add content_access_by_path to the default install of LGD? 

I haven't done that yet. Do we want it as part of the default install?

**Comment by willguv** [comment]
I'm thinking yes part of the default install but maybe switched off?

**Comment by markconroy** [comment]
PR for this: https://github.com/localgovdrupal/localgov_core/pull/215

**Comment by willguv** [comment]
We've agreed to not include this by default, but document it. I'll find the related card and close this one

**Comment by Dixonj1** [comment]
@markconroy we've tested this and authors can still publish new content anywhere on the site, even if they're restricted to editing certain sections of it. Is that right?

**Comment by markconroy** [comment]
@Dixonj1, yes, I don't see why we would stop them from placing content somewhere.

I guess we could have a checkbox on the settings page to say "Only allow editors to save content within the paths they are allowed to edit". I'm not fully-sure how that would work, but I could give it a try.

**Comment by Dixonj1** [comment]
@markconroy that would be really useful for us, thanks. If we're restricting author access we need to be able to stop them publishing content.

**Comment by markconroy** [comment]
Issue created - https://www.drupal.org/project/content_access_by_path/issues/3479092

**Comment by Dixonj1** [comment]
@markconroy we've discovered that if an author has created a page, this overrides restrictions through content access by path. If the piece of content they created is outside of their restricted content, they can still edit it. It would be worth letting people know this in the documentation.

**Comment by markconroy** [comment]
@Dixonj1 That is by design. Editors can edit

1. Items within their allowed paths, and
2. Items that they are the author of

The second part is deliberate. If not, and they create a piece of content with the wrong path by accident, they would not be able to then edit it to set the right path.

**Comment by willguv** [comment]
@willguv to add an extra page to menu in this section: https://docs.localgovdrupal.org/content/features/subsites-extras.html

https://www.drupal.org/project/content_access_by_path

**Comment by willguv** [comment]
PR here https://github.com/localgovdrupal/docs/pull/303
